test/assertions_test.lua

```diff
@@ -3,6 +3,13 @@ local g = t.group()
 
 local helper = require('test.helper')
 
+g = t.group('pgroup', {{a = 1, b = 4}, {a = 2, b = 3}})
+
+g.test_params = function(cg)
+    print('a = %s', cg.params.a)
+    print('b = %s', cg.params.b)
+end
+
```

Comment:
It seems, it is forgotten snippet for debugging.

Reply:
Thanks for pointing it out! Fixed.